docs/verify/simulate-with-qiskit-aer.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<p>I think at this point we should link to the Qiskit Aer api reference: <a href="https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Estimator.html#estimator" target="_blank">https://qiskit.org/ecosystem/aer/stubs/qiskit_aer.primitives.Estimator.html#estimator</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/302/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done.